File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.js
function solution(numbers) {
    let answer = 0;
    let max1=0;
    let max2=0;
    let index=0;
    for(let i=0; i<numbers.length ; i++){  
        if(numbers[i]>=max1) {
            max1=numbers[i];
            index=i;
        }
    }
    for(let i=0; i<numbers.length; i++){
        if(numbers[i]>=max2 && i!=index ) {
            max2=numbers[i];
        }                             
    }
    answer=max1*max2;
    return answer;
}

File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers) {
    int answer = 0;
    int max1=0;
    int max2=0;
    int index=0;
    for(int i=0; i<size(numbers); i++){  
        if(numbers[i]>=max1) {
            max1=numbers[i];
            index=i;
        }
    }
    for(int i=0; i<size(numbers); i++){
        if(numbers[i]>=max2 && i!=index ) {
            max2=numbers[i];
        }                             
    }
    answer=max1*max2;
    return answer;
}